base/bitset.jl

```diff
@@ -68,14 +68,14 @@ function _bits_findnext(b::Bits, start::Int)
     # start is 0-based
     # @assert start >= 0
     _div64(start) + 1 > length(b) && return -1
-    unsafe_bitfindnext(b, start+1) - 1
+    zero_sentinel(unsafe_bitfindnext(b, start+1)) - 1
```

Comment:
Why not keep `unsafe_bitfindnext` returning `0` for no match, since it's internal? Wouldn't that involve less changes?

Reply:
It's one of each; I decided to change it for greater consistency. But indeed I followed this advice for other internal methods.